docs/docs/configuration/reference.md

```diff
@@ -712,6 +712,18 @@ cameras:
       # By default the cameras are sorted alphabetically.
       order: 0
 
+    # Optional: Configuration for AI generated tracked object descriptions
+    genai:
```

Comment:
better to not include there here and instead just put that it can be overridden at the camera level

Reply:
Only thing is that the options are different at the camera level, you can't have `model`, `provider`, etc that the root level has. So I just copied what we did for birdseye, which is also in the reference at the camera level.

Reply:
I don't see birdseye referenced at the camera level

Reply:
https://github.com/blakeblackshear/frigate/blob/0dc72173f4cfd49a044db33df4c16cc7887839ce/docs/docs/configuration/reference.md?plain=1#L710

Reply:
lol, the mark for control + f was hidden by an overlay. 👍